service/tablet_allocator.cc

```diff
@@ -1509,6 +1518,15 @@ class load_balancer {
                 nodes[dst.host].target_load_sketch->unload(dst.host, dst.shard);
             });
 
+            if (lblogger.is_enabled(seastar::log_level::trace)) {
+                shard_id shard = 0;
+                for (auto&& shard_load : target_info.shards) {
+                    lblogger.trace("shard {}: all tablets: {}, candidates: {}, tables: {}", tablet_replica {dst.host, shard},
+                                   shard_load.tablet_count, shard_load.candidate_count(), shard_load.tablet_count_per_table);
+                    shard++;
+                }
+            }
+
```

Comment:
We should have a REST API + nodetool command for such things.

Reply:
Yes. But logging is useful too, for analyzing evolution of state.